frontend/src/metabase/home/containers/SearchApp.jsx

```diff
@@ -48,12 +47,13 @@ export default class SearchApp extends React.Component {
                     .groupBy("model")
                     .pairs()
                     .value()
+                    .reverse() // show dashboards and collections before questions
```

Comment:
`.groupBy("model")` seems to result in an ordering that you're then reversing. Can we just be explicit about the order of the types (as defined by `model`)? 

Feel like this is a little too cute